examples-gallery/plot_ball_rolling_on_spinning_disc.py

```diff
@@ -361,25 +363,25 @@ def obj_grad(free):
 i4 = np.zeros(8*num_nodes)
 initial_guess = np.hstack((i1,i1a, i1b, i2, i3, i4, 0.01))
 
-fig1, ax1 = plt.subplots(14, 1, figsize=(7.25, 0.75*14), sharex=True,
-    layout='constrained')
-prob.plot_trajectories(initial_guess, ax1)
-
```

Comment:
If you delete a figure, then the thumbnail figure may change, have you checked that? There will no longer by a figure 7, right?

I don't see any reason to delete the figure.

Reply:
> If you delete a figure, then the thumbnail figure may change, have you checked that? There will no longer by a figure 7, right?
> 
> I don't see any reason to delete the figure.

I did not intentionally delete this. No idea what happened.